pytest_odoo.py

```diff
@@ -78,7 +78,7 @@ def _importtestmodule(self):
             modname = ".".join(names)
             # for modules in openerp/addons, since there is a __init__ the
             # module name is already fully qualified (maybe?)
-            if not modname.startswith(odoo_namespace + '.addons.'):
+            if not modname.startswith(odoo_namespace + '.addons.') and odoo.release.version_info[0] > 8:
```

Comment:
Well, the point of this branch is to prefix odoo modules names (e.g. `sale`,  `sale_stock`, ...) with `odoo.addons.` (e.g. `odoo.addons.sale`, `odoo.addons.sale_stock`, ...) (or `openerp.addons` for 9.0 and 8.0).

If you ignore it for version 8.0, it means the modules will be imported in the main global namespace. Generally this is not so much of an issue, as odoo looks for them both in the global namespace and in `odoo/openerp.addons` (for backward compatibility reasons). But some community addons (such as connector) rely on the modules path to be the full one, so their tests would not pass.

Could you find what was failing exactly?